src/common-rest-api/src/main/resources/common-openapi-definition.yaml

```diff
@@ -17,20 +17,32 @@ servers:
 tags:
   - name: dummy
```

Comment:
tags here, and in proxy-ui-api/openapi-definition, have some issues. 

common-openapi-definition endpoint uses tag "user", which is not introduced
proxy-ui-api/openapi-definition introduces tag "user", which is not used in any endpoint

Reply:
Fixed the tags